src/alignment/model_utils.py

```diff
@@ -96,5 +96,11 @@ def get_peft_config(model_args: ModelArguments) -> PeftConfig | None:
 
 
 def is_adapter_model(model_name_or_path: str, revision: str = "main") -> bool:
-    repo_files = list_repo_files(model_name_or_path, revision=revision)
-    return "adapter_model.safetensors" in repo_files or "adapter_model.bin" in repo_files
+    try:
```

Comment:
I think we can do something a bit cleaner as follows:

```python
from huggingface_hub.utils._validators import HFValidationError

def is_adapter_model(model_name_or_path: str, revision: str = "main") -> bool:
    try:
        # Try first if model on a Hub repo
        repo_files = list_repo_files(model_name_or_path, revision=revision)
    except HFValidationError:
        # If not, check local repo
        repo_files = os.listdir(model_name_or_path)
    return "adapter_model.safetensors" in repo_files or "adapter_model.bin" in repo_files
```

Reply:
Thanks @lewtun, I like your cleaner solution. I will replace it and run `make style && make quality`